src/HeaderRow.tsx

```diff
@@ -33,13 +33,13 @@ const headerRow = css`
 
   > .${cell} {
     /* Should have a higher value than 1 to show up above frozen cells */
```

Comment:
The comment doesn't make sense anymore since we use a single `grid` now
```suggestion
    /* Should have a higher value than 0 to show up above regular cells */
```